docs/configuration/index-config.md

```diff
@@ -5,12 +5,13 @@ sidebar_position: 2
 
 This page describes how to configure an index.
 
-In addition to the `index_id`, the index configuration lets you define five items:
+In addition to the `index_id`, the index configuration lets you define six items:
```

Comment:
Just noticing that this should probably still be 5 items?  Was this an error left over from previous changes?